backend/jobs/processor.js

```
@@ -9,29 +9,31 @@ const { routeToChannel } = require('../services/channelRouter');
-        }
-        return value;
-      }, 2);
-      console.log(`📊 JOB ${job.id} DATA:`, cleanData);
-    } catch (logError) {
-      console.log(`📊 JOB ${job.id} DATA: [Unable to stringify safely]`, Object.keys(data));
+    // Only log full data in development
+    if (process.env.NODE_ENV === 'development') {
+      try {
+        const cleanData = JSON.stringify(data, (key, value) => {
+          if (typeof value === 'string') {
+            return value.replace(/[\x00-\x1F\x7F-\x9F]/g, '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)